### ISSUE TITLE
[Bug] Posts with markup does not render correctly in dark mode

### ISSUE THREAD

**Issue by austinhuang0131** [ZeeRooo/MaterialFBook#220]
v4.0.0. Posts without markup (bold, italics, etc) render in the light gray color correctly, but posts with markup remain in black, making them unreadable.

**Comment by ZeeRooo** [comment]
Can you post an example? I see no issues here (i don't know if you are talking about this kind of bold text)
![Capture+_2020-04-06-02-43-39](https://user-images.githubusercontent.com/19499008/78526535-ad14ea80-77b0-11ea-932a-e8a0a0935018.png)

**Comment by austinhuang0131** [comment]
![sketch-1586178933471](https://user-images.githubusercontent.com/16656689/78562379-3fd07c00-77e7-11ea-93fe-b053f7a6db46.png)

As you can see, the post itself contains italics.

**Comment by austinhuang0131** [comment]
Problem persists in new release

![Screenshot_20200414-010607](https://user-images.githubusercontent.com/16656689/79188111-297c7e80-7dec-11ea-8f0e-5e8c7dcb2560.png)

**Comment by ZeeRooo** [comment]
I can't replicate the issue. Can you share any post/page with this issue?

**Comment by austinhuang0131** [comment]
https://m.facebook.com/notes/association-pour-la-mobilit%C3%A9-active-du-canal-de-lachine/lama-du-canal-de-lachine-applaudit-les-mesures-temporaires-de-s%C3%A9curisation-pi%C3%A9to/3008377112518129/?ref=group_browse

**Comment by ZeeRooo** [comment]
Please try with the newer version

**Comment by austinhuang0131** [comment]
Yeah, that example seems to be fixed, but not the markup issue. Here's a post under a certain Chrome dark mode extension:

![1587652736953](https://user-images.githubusercontent.com/16656689/80112224-decad700-854e-11ea-87b1-90d985661a09.jpg)

Here's what it looks like in the app:

![Screenshot_20200423-103417](https://user-images.githubusercontent.com/16656689/80112287-ed18f300-854e-11ea-9e41-fb774bbdf5a3.png)

https://m.facebook.com/groups/1127887950893752/ (had to make a testing group to cover all markup cases)

Also the username is black for like all posts (with or without markup)

**Comment by PanderMusubi** [comment]
See also https://github.com/ZeeRooo/MaterialFBook/issues/144

But, is this app still being maintained?
